css/_mixins.scss

```diff
@@ -92,7 +92,14 @@
     }
 }
 
-
+@mixin button () {
+    .button {
+        background:none;
+        border:none;
+        margin:0;
+        padding:0;
+    }
+}
```

Comment:
Please add a new blank line between this and next line, and add a space between properties and values above.